visualizer/visualize-trust.js

```diff
@@ -0,0 +1,63 @@
+const { Client } = require('pg');
```

Comment:
It is possible to run the script from the shell directly through the interpreter. The `#!` (Shebang) notation can be applied as of:

```suggestion
#!/usr/bin/env node
const { Client } = require('pg');
```

Then when adding

- [ ] `chmod u+x`

to the file, it can be invoked with `./visualizer/visualize-trust.js`

Removing the file extension altogether renders this into a well-known form, such as `./visualizer/visualize-trust`.